bundle-workflow/README.md

```diff
@@ -216,3 +275,24 @@ The following options are available.
 |--------------------|-------------------------------------------------------------------------|
 | --keep             | Do not delete the temporary working directory on both success or error. |
 | -v, --verbose      | Show more verbose output.                                               |
+
+### Component Onboarding
+
+1. Update a [manifest](/manifests) for a particular release to include your plugin.  For example to be included in the 1.1.0 release, you would update [opensearch-1.1.0.yml](/manifests/1.1.0/opensearch-1.1.0.yml). 
+   We require your plugin name, repository url, and git ref
+   that should be used. For unreleased versions this should be a branch in your repository.  Once a release is cut, these refs will be updated
+   to build from a tag or specific commit hash.
+
+2. Create a build.sh if you have specific requirements that are not covered by the [default build.sh script](/scripts/default/build.sh) and commit it to your repository. It
```

Comment:
It should live in `scripts`, let's not say root of repo (I don't think anyone actually has one in root of repo and we should get rid of this).